x-pack/legacy/plugins/siem/public/components/page/network/network_top_countries_table/index.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiFlexItem } from '@elastic/eui';
+import { isEqual, last } from 'lodash/fp';
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { ActionCreator } from 'typescript-fsa';
+import { StaticIndexPattern } from 'ui/index_patterns';
+
+import { networkActions } from '../../../../store/actions';
+import {
+  Direction,
+  FlowTargetSourceDest,
+  NetworkTopCountriesEdges,
+  NetworkTopNFlowFields,
+  NetworkTopNFlowSortField,
+} from '../../../../graphql/types';
+import { networkModel, networkSelectors, State } from '../../../../store';
+import { Criteria, ItemsPerRow, PaginatedTable } from '../../../paginated_table';
+
+import { getCountriesColumnsCurated } from './columns';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: NetworkTopCountriesEdges[];
+  fakeTotalCount: number;
+  flowTargeted: FlowTargetSourceDest;
+  id: string;
+  indexPattern: StaticIndexPattern;
+  ip?: string;
+  isInspect: boolean;
+  loading: boolean;
+  loadPage: (newActivePage: number) => void;
+  showMorePagesIndicator: boolean;
+  totalCount: number;
+  type: networkModel.NetworkType;
+}
+
+interface NetworkTopCountriesTableReduxProps {
+  activePage: number;
+  limit: number;
+  topCountriesSort: NetworkTopNFlowSortField;
```

Comment:
If the pattern is to re-use these `SortField` types we may want to rename them so they are generic and not tied to a specific implementation .